src/functions/txc-processor/data/trips.ts

```diff
@@ -46,7 +35,7 @@ export const processTrips = async (
                 vehicle_journey_code: vehicleJourney.VehicleJourneyCode,
                 ticket_machine_journey_code: vehicleJourney.Operational?.TicketMachine?.JourneyCode || "",
                 file_path: filePath,
-                direction: getGtfsDirectionId(journeyPattern?.Direction),
+                direction: getDirectionRef(journeyPattern?.Direction),
```

Comment:
Will this change what ends up in the database? We probably want to keep that the same so it doesn't break the SIRI-VM generation